src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import HeaderComponent from "./component/Header-section/header.js";
+import Header from "./component/Header-section/Header.css";
```

Comment:
CSS file doesn't have to be imported as a named import. And it's being used as a component here, which is wrong.
So import the CSS file in its appropriate component.